src/features/common/Layout/UserPropsContext.tsx

```diff
@@ -136,6 +140,8 @@ export const UserPropsProvider: FC = ({ children }) => {
     auth0User: user,
     auth0Error: error,
     loadUser,
+    refetchData,
+    setRefetchData,
```

Comment:
I wonder if keeping `refetchData` / `setRefetchData` here makes the relevant code a bit harder to read.

Reply:
Let's just rename this to `refetchUserData` and `setRefetchUserData` for now.